.changeset/thin-carrots-shop.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Added: `no-descending-specificity` line number to the referenced selector in message
```

Comment:
```suggestion
Added: line number of the anchoring selector to `no-descending-specificity`'s report message
```

It seems better to me; @ybiquitous what do you think?

Reply:
@Mouvedia Your suggested message might be more accurate, but let's keep the format "*Added: `rule-name` ...*". The current one is sufficient.

Reply:
I don't care if we don't use mine, but the current one makes no sense.
i.e. it's not adding the line number of `no-descending-specificity`
It needs to be emended.

Reply:
Is this option acceptable?

Added: `no-descending-specificity` report message includes line number of the anchoring selector

Reply:
> Is this option acceptable?
> 
> Added: `no-descending-specificity` report message includes line number of the anchoring selector

Sounds good 👍🏼

Reply:
Done